src/system/Footer/Footer.test.tsx

```diff
@@ -0,0 +1,48 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { Footer } from './Footer';
+
+const links = [
+	{
+		children: 'Link1',
+		href: 'https://wpvip.com/',
+	},
+	{
+		children: 'Link2',
+		href: 'https://docs.wpvip.com/',
+	},
+];
+
+describe( '<LinkExternal />', () => {
+	it( 'should render LinkExternal props', () => {
+		const moreLinks = [
+			{
+				children: 'Link3',
+				href: 'https://wpvipstatus.com',
+				newTab: true,
+			},
+		];
+
+		const combinedLinks = [ ...links, ...moreLinks ];
+
+		render( <Footer links={ combinedLinks } /> );
+
+		const link1 = screen.getByRole( 'link', { name: /link1/i } );
+		const link3 = screen.getByRole( 'link', { name: /link3/i } );
+
+		expect( link1 ).toHaveAttribute( 'target', '_self' );
+		expect( link3 ).toHaveAttribute( 'target', '_blank' );
+	} );
+
+	it( 'should hide last separator when hasTrailingSeparator is true', () => {
```

Comment:
Can you make this work with CSS using `last-child`? On the Breadcrumb components, we use the separators using CSS ::before.

Ex: https://github.com/Automattic/vip-design-system/blob/trunk/src/system/Nav/styles/variants/breadcrumbs.ts#L11

Reply:
Thanks for the example! I'll update to be more consistent with that 🙂

Reply:
https://github.com/Automattic/vip-design-system/blob/6b6cc61a518d088b67d4b915fa21c0108006e958/src/system/Breadcrumbs/Breadcrumbs.tsx#L151-L155

@djalmaaraujo I was trying to replicate this but I'm ending up with `[Object Object]` as my class name in the dom. 😅  When looking into it, I see mentions that `sx` only works on custom components. Curious on how you made it work!

Reply:
Solved here: https://github.com/Automattic/vip-design-system/pull/409/commits/70cd4234dd163c7f6a9640e1352274d01e8e34e9